### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by patcon** [widgit-cookbooks/nxlog#12]
Can we get a new release (tagged and pushed to supermarket) to go with the nxlog version bump?

If you're not using it, `stove` makes releasing so simple that it's a joke
https://github.com/sethvargo/stove

**Comment by gh2k** [comment]
Thanks. I've been trying to get the integration tests to pass. I think that they work, but the CI box is telling me
```
[WinRM::Transport::FileTransporter] Upload failed (exitcode: 0), but stderr present
```
when trying to run the Windows tests under `kitchen`. I may just rely on manual tests and push it anyway, but I'm going to spend a little more time on it first.

Incidentally, it's really annoying that nxlog remove old versions as new ones go up, as it immediately breaks the cookbook. I'm considering mirroring the files to stop this from happening.

**Comment by patcon** [comment]
Ah, gotcha. No rush! :)

**Comment by gh2k** [comment]
This is now released as 0.8.6. Sorry for the delay... Had some Fun making it play nice with Chef 12.5 on Windows. I've set up a mirror for the nxlog binaries so that the cookbook doesn't break next time they release a new version.

**Comment by patcon** [comment]
Thanks @gh2k!
